docs/eventing/brokers/broker-types/kafka-broker/README.md

```diff
@@ -386,6 +387,56 @@ The supported consumer delivery guarantees are:
 
 `unordered` is the default ordering guarantee.
 
-### Additional information
+## Data plane Isolation vs Shared Data plane
```

Comment:
I'd expand with a sentence like "if you don't what to chose go with `Kafka` unless you have specific isolation requirements"